sass/_roster.scss

```diff
@@ -119,7 +119,9 @@
 
                     .avatar {
                         float: left;
```

Comment:
We're using flexbox for many things now, so this `float: left` statement might no longer be necessary.

Reply:
Yes. In fact, I found it here, and when I tried to remove it, it broke the design. I'll try to remove it again and fix the display with flexbox.